core/src/test/java/greencity/security/filters/AccessTokenAuthenticationFilterTest.java

```diff
@@ -44,13 +50,17 @@ public void setUp() {
 
     @Test
     public void doFilterInternal() throws IOException, ServletException {
+        Optional<User> user = Optional.of(ModelUtils.getUser());
+        List<GrantedAuthority> authorities = List.of(new SimpleGrantedAuthority("ROLE_USER"));
```

Comment:
I was expecting to see 

- `greencity.entity.enums.ROLE` as input parameter
- `List<GrantedAuthority>` as output parameter checked somewhere in assertions

to make sure that the conversion was correct.